control-plane-operator/controllers/hostedcontrolplane/config/deployment.go

```diff
@@ -195,3 +195,15 @@ func (c *DeploymentConfig) ApplyToDaemonSet(daemonset *appsv1.DaemonSet) {
 	c.Resources.ApplyTo(&daemonset.Spec.Template.Spec)
 	c.AdditionalAnnotations.ApplyTo(&daemonset.Spec.Template.ObjectMeta)
 }
+
+func (c *DeploymentConfig) ApplyToStatefulSet(sts *appsv1.StatefulSet) {
```

Comment:
Just a note for posterity... `WorkloadConfig` is probably a better name for this struct (instead of `DeploymentConfig`)

Reply:
I just want to not have something named `DeploymentConfig` :-p

Reply:
but definitely don't need to address it in this PR

Reply:
I support this, I also thought this was about an Openshift Deploymentconfig when I first saw this in code...But yeah unrelated to this PR